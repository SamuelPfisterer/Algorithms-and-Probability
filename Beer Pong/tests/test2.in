30
2 13 3 2
0.510731 0.902326
2 17 15 2
0.568095 0.714456
2 12 19 2
0.582898 0.824514
2 2 2 2
0.645346 0.998567
2 5 11 2
0.750274 0.660494
2 5 9 2
0.655779 0.575381
2 2 6 2
0.758823 0.971549
2 11 8 2
0.317903 0.823602
2 10 15 2
0.975279 0.877182
2 7 16 2
0.862975 0.803745
2 10 13 2
0.578379 0.670702
2 2 14 2
0.3363 0.905034
2 9 12 2
0.796939 0.612516
2 9 20 2
0.325051 0.903139
2 6 18 2
0.754916 0.778947
2 13 14 2
0.568307 0.812953
2 11 18 2
0.57117 0.869851
2 9 9 2
0.38797 0.848771
2 18 10 2
0.372772 0.623302
2 2 6 2
0.863115 0.986774
2 12 10 2
0.391868 0.729083
2 17 6 2
0.88649 0.690614
2 5 11 2
0.464507 0.794757
2 5 9 2
0.994137 0.629646
2 15 18 2
0.355369 0.590704
2 18 6 2
0.663835 0.860468
2 7 4 2
0.487297 0.811619
2 18 3 2
0.338492 0.624861
2 6 14 2
0.537676 0.865947
2 11 15 2
0.916771 0.975524
