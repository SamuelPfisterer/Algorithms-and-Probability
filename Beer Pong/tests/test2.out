0.42037
0.2656756
0.2141484
0.4300118
0.2216407
0.2011301
0.2152861
0.1695102
0.4049645
0.2656933
0.2178114
0.0419205
0.3177986
0.0974281
0.1841283
0.2584799
0.210859
0.1852002
0.1802044
0.2461527
0.1826504
0.6518669
0.1360249
0.3810463
0.1151473
0.4926668
0.3048325
0.2180474
0.158452
0.4025386
